setup.py

```diff
@@ -42,7 +42,8 @@
     maintainer_email='support@datarobot.com',
     license='BSD',
     url='http://www.datarobot.com/',
-    packages=['datarobot_batch_scoring'],
+    packages=['datarobot_batch_scoring',
+              'datarobot_batch_scoring.network'],
```

Comment:
I'd suggest to use [`setuptools.find_packages()`](http://setuptools.readthedocs.io/en/latest/setuptools.html#using-find-packages) in order to prevent similar issues in the future. It simply recursively walks and find packages to install.

E.g.:

```python
from setuptools import setup, find_packages


packages=find_packages(exclude=['tests*'])
```